frontend/src/js/controllers/authCtrl.js

```diff
@@ -163,7 +164,7 @@
                                     vm.FormError = response.data.non_field_errors[0];
                                 }
                             } catch (error) {
-                                console.log(error);
```

Comment:
Please replace `Your data is invalid.` with `Some error occurred. Please try again later. `

Reply:
Ok. I will send a commit fix the message.

Reply:
Ready. 👍